recipe/meta.yaml

```diff
@@ -26,8 +26,8 @@ requirements:
     - wheel
   run:
     - python
-    - protobuf >=3.19.4
-    - types-protobuf >=3.19.12
+    - protobuf >=4.25.3
```

Comment:
do we need an upper bound on this and/or using the libprotobuf pinning from the base cbc?

Reply:
It's the protobuf package (python), not libprotobuf. So this is fine.